src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -175,17 +174,6 @@ public static SimulatedPageFile MakeRealPublishModeFileInBookFolder(HtmlDom dom)
 			return new SimulatedPageFile() { Key = uri.AbsoluteUri };
 		}
 
-		private static string RemoveLocalhostReferenceFromPath(string path)
-		{
-			if (String.IsNullOrEmpty(path) || !path.StartsWith(PathEndingInSlash))
-				return path;
-			path = path.Substring(PathEndingInSlash.Length - 1);
-			path = Regex.Replace(path, "^/([A-Z])%3A/", "$1:/");
```

Comment:
Seems we might have lost something here...as well as unescaping the %3A, this was removing the leading slash?